DXRModules/DeusEx/Classes/DXREventsBase.uc

```diff
@@ -1339,6 +1339,8 @@ function _MarkBingo(coerce string eventname)
         class'DXRTelemetry'.static.SendEvent(dxr, player(), j);
 
         CheckBingoWin(dxr,nowbingos);
+
+        DXRBingoCampaign(dxr.FindModule(class'DXRBingoCampaign', true)).HandleBingo(nowbingos);
```

Comment:
Could do this inside the bingo win so that it's based on the bingo_win flag and can be customized

Reply:
Yeah, I like that. I wanted it to eventually be based on that setting.